javascript/ql/src/Security/CWE-079/ExceptionXss.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name Client-side cross-site scripting through exception
+ * @description User input being part of an exception allows for 
```

Comment:
I find this description difficult to parse, but I am not a developer. Consider re-wording, or just ignore me if you think users (developers) will understand.

Reply:
I changed a tiny bit of wording in the description, but that wasn't related to your comment.   
The current wording is: 
```
User input being part of an exception allows for 
cross-site scripting if that exception is written
to the DOM.  
```

I also find it long, but I need to make sure it sets itself apart from `Xss.ql`, and can't figure out a better alternative.

Reply:
"Inserting data from an exception containing user input into the DOM may enable cross-site scripting."?